package.json

```diff
@@ -91,6 +91,7 @@
     "bs58": "^5.0.0",
     "commander": "^12.1.0",
     "ethers": "^6.12.0",
+    "eventemitter3": "^5.0.1",
```

Comment:
more recent node polyfills may provide support for EventEmitter and we already require the node polyfill given our us of `fs`, `buffer`, `crypto`, etc.

that said - requiring the node polyfill has proven to be a pain point for builders so we can leave this for now and draft notes on how to handle those other imports

Reply:
Agreed - uncertain on the stream polyfill. I reviewed the options there and it doesnt seem wholely straightforward